README.md

```diff
@@ -671,3 +671,91 @@ metric-config.<metricType>.<metricName>.<collectorType>/interval: "30s"
 
 The default is `60s` but can be reduced to let the adapter collect metrics more
 often.
+
+## ScalingSchedule Collectors
+
+The `ScalingSchedule` and `ClusterScalingSchedule` collectors allow
+collecting time-based metrics from the respective CRD objects specified
+in the HPA.
+
+### Supported metrics
+
+| Metric | Description | Type | K8s Versions |
+| ---------- | -------------- | ------- | -- |
+| ObjectName | The metric is calculated and stored for each `ScalingSchedule` and `ClusterScalingSchedule` referenced in the HPAs | `ScalingSchedule` and `ClusterScalingSchedule` | `>=1.12` |
+
+### Example
+
+This is an example of using the ScalingSchedule collectors to collect
+metrics from a deployed kind of the CRD. First, the schedule object:
+
+```yaml
+apiVersion: zalando.org/v1
+kind: ClusterScalingSchedule
+metadata:
+  name: "scheduling-event"
+spec:
+  schedules:
+  - type: OneTime
+    date: "2021-10-02T08:08:08+02:00"
+    durationMinutes: 30
+    value: 10
+  - type: Repeating
+    durationMinutes: 10
+    value: 12
+    period:
+      startTime: "15:45"
+      timezone: "Europe/Berlin"
+      days:
+      - Mon
+      - Wed
+      - Fri
+```
+
+This resource defines a scheduling event named `scheduling-event` with
+two schedules of the kind `ClusterScalingSchedule`.
+
+`ClusterScalingSchedule` objects aren't namespaced, what means it can be
+referenced by any HPA in any namespace in the cluster. `ScalingSchedule`
+have the exact same fields and behavior, but can be referenced just by
+HPAs in the same namespace. The schedules can have the type `Repeating`
+or `OneTime`.
+
+Check the CRDs definitions in the `docs` directory for a better
+understanding of the possible fields and their behavior.
+
+An HPA can reference the deployed `ClusterScalingSchedule` object as
+this example:
+
+```yaml
+apiVersion: autoscaling/v2beta2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: "myapp-hpa"
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: myapp
+  minReplicas: 1
+  maxReplicas: 15
+  metrics:
+  - type: Object
+    object:
+      describedObject:
+        apiVersion: zalando.org/v1
+        kind: ClusterScalingSchedule
+        name: "scheduling-event"
+      metric:
+        name: "scheduling-event"
+      target:
+        type: AverageValue
+        averageValue: "1"
```

Comment:
We should probably explain that in this example it would mean that it scales to `12` in the period defined, just to make it obvious what the relation is.

Reply:
Yeah, makes sense, a good catch.